### ISSUE TITLE
psplash at boot time is not shown

### ISSUE THREAD

**Issue by torrentalle** [jumpnow/meta-bbb#2]
Hi,
Yocto psplash in not shown at boot time.  During shutdown is shown and I can launch manually.
How can I enbale it?

**Comment by scottellis** [comment]
The default psplash recipe starts the psplash script before
file systems are mounted.

root@bbb:~# ls -l /etc/rcS.d
total 0
lrwxrwxrwx 1 root root 20 Feb 23 17:56 S00psplash.sh -> ../init.d/psplash.sh
lrwxrwxrwx 1 root root 19 Jan 30 15:47 S02banner.sh -> ../init.d/banner.sh
lrwxrwxrwx 1 root root 18 Jan 30 15:47 S02sysfs.sh -> ../init.d/sysfs.sh
lrwxrwxrwx 1 root root 21 Jan 30 15:47 S03mountall.sh -> ../init.d/mountall.sh
lrwxrwxrwx 1 root root 23 Feb 23 17:56 S03systemd-udevd -> ../init.d/systemd-udevd
lrwxrwxrwx 1 root root 21 Feb 23 17:56 S05modutils.sh -> ../init.d/modutils.sh
...

I added a psplash.bbappend to meta-bbb/recipes-core to fix this.

root@bbb:~# ls -l /etc/rcS.d
total 0
lrwxrwxrwx 1 root root 19 Jan 30 15:47 S02banner.sh -> ../init.d/banner.sh
lrwxrwxrwx 1 root root 18 Jan 30 15:47 S02sysfs.sh -> ../init.d/sysfs.sh
lrwxrwxrwx 1 root root 21 Jan 30 15:47 S03mountall.sh -> ../init.d/mountall.sh
lrwxrwxrwx 1 root root 23 Feb 23 17:56 S03systemd-udevd -> ../init.d/systemd-udevd
lrwxrwxrwx 1 root root 20 Feb 23 17:56 S04psplash.sh -> ../init.d/psplash.sh
lrwxrwxrwx 1 root root 21 Feb 23 17:56 S05modutils.sh -> ../init.d/modutils.sh
...

I also added psplash to the console-image.

The changes are pushed to meta-bbb [jethro] branch.
